otx/algorithms/detection/configs/detection/mobilenetv2_ssd/model.py

```diff
@@ -23,16 +23,27 @@
 
 __width_mult = 1.0
 
+runner = dict(max_epochs=20)
+
 model = dict(
     type="CustomSingleStageDetector",
     bbox_head=dict(
         type="CustomSSDHead",
         num_classes=80,
         in_channels=(int(__width_mult * 96), int(__width_mult * 320)),
+        use_depthwise=True,
+        norm_cfg=dict(type="BN", eps=0.001, momentum=0.03),
+        act_cfg=dict(type="ReLU"),
+        init_cfg=dict(type="Normal", layer="Conv2d", std=0.001),
         anchor_generator=dict(
             type="SSDAnchorGeneratorClustered",
             strides=(16, 32),
-            reclustering_anchors=True,
+            # FIXME: Add this feature to mpa.det.inferrer
+            # This feature is implemented in mpa.det.trainer but not in mpa.det.inferrer
+            # To fix this, we need DetectionInferenceTask.infer in otx
+            # to take whole dataset, not validation subset only.
+            # This fix could impact other tasks infer method
+            reclustering_anchors=False,
```

Comment:
We don't need to re-cluster anchors in infer/eval stage as we just need to adjust anchor settings from the training set before train start. Could you explain why you disabled this?

Reply:
Since this feature is not implemented in `mpa.det.inferrer`, when doing inference, anchors is not reclustered resulting hard-to-debug error.
Inference performance is a disaster since anchors are not the same when training and inference.

Reply:
I thought that clustered anchor setting is used in inference time as is.
@jaegukhyun Could you give some advice here?

Reply:
@goodsong81 @jaegukhyun This has been fixed by https://github.com/openvinotoolkit/training_extensions/pull/1316/commits/92c551f63a0f31dcbb8e6d4326a7192091e8a87f